torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1581,7 +1602,7 @@ def call_function(
             proxy=tx.output.create_proxy(
                 "call_function",
                 self.value,
-                args=inp_args + p_args,
+                args=inp_args[:3] + p_args[:1] + inp_args[3:] + p_args[1:],
```

Comment:
Why does this need to be done?

Reply:
because ```p_args``` are from ```create_wrapped_node``` which generates subgraph body node and lifted args, we should split them to follow the particular order of the HOO's arguments.

Reply:
can you write a comment about this, since this interleaving is pretty confusing

Reply:
done